lib/web_ui/build.html.yaml

```diff
@@ -9,6 +9,7 @@ targets:
           compiler: dart2js
           dart2js_args:
             - --no-minify
+            - --disable-inlining
```

Comment:
This option only applies to engine unit-tests, where performance doesn't matter (we also enable assertions). I thought it would be more useful for tests to output good stack traces when they fail. I lost some time chasing an issue that was obscured by missing stack frames that were removed by inlining.

Reply:
thx. makes sense in tests.